app/pages/Settings/index.js

```diff
@@ -245,6 +250,12 @@ export default class Settings extends Component {
                 'Remove Wallet ',
                 () => history.push('/settings/wallet/new-wallet'),
               )}
+              {this.renderSection(
+                'API Key',
+                'API key for hsw-cli and hsw-rpc. Please make sure you select the wallet "allison"',
```

Comment:
Can we find a way to link [this](https://hsd-dev.org/api-docs/#selectwallet)?   Otherwise unlikely to really inform anyone.

The users we're trying to protect are unlikely to know what it means to select a wallet, so official API docs may help them get up to speed.  Other users won't know what to make of "Allison" - sounds like it could be a setting in Bob.

Reply:
The api-docs intro section has a bit better explanation under the "Wallet: rpc selectwallet vs --id=..." section up top.

I think a CLI guide in general might be helpful, maybe even as a complete built in page in Bob UI. We talked about this before too, about having a built-in console entirely inside Bob, like electrum:

https://bitzuma.com/posts/an-introduction-to-the-electrum-python-console/

Reply:
Just want to say ditto for Electrum style console for CLI and logs inside Bob. It would be a really nice addition and provide a clean way to directly use the node as fallback. In terms of UI it can simply be a tab in Settings.